frontend/src/toolbar/stats/HeatmapToolbarMenu.tsx

```diff
@@ -75,7 +75,7 @@ const MenuBody = (): JSX.Element => {
                     />
                 </div>
             </Tooltip>
-            <div className="FlagList flex flex-col w-full overflow-y-scroll h-full">
+            <div className="FlagList flex flex-col w-full h-full">
```

Comment:
I preferred these menus with a separate not-scrolling header but would rather get it all merged than not 🤷 

![2023-11-17 13 38 52](https://github.com/PostHog/posthog/assets/984817/e8090e00-0cc9-4fd6-ae8e-99fa0f75aa61)

Reply:
I do to but it was getting tough to get right on a smaller screen with all the possible flexing that can happen